core/spec/models/spree/variant_spec.rb

```diff
@@ -773,6 +773,42 @@
     end
   end
 
+  describe '#shipping_category' do
+    context 'when shipping_category is nil' do
+      let(:product) { build(:product) }
```

Comment:
I'd explicitly add a shipping category to the product. Otherwise, if it were `nil` and the tested code didn't delegate to the product, we would have a false positive.

Reply:
Fair point. Adding